resolver/contentResolver.go

```diff
@@ -57,7 +57,7 @@ func (cr *defaultContentResolver) callContentResolverApp(diffUuids map[string]bo
 	req.Header.Set("Content-Type", "application/json; charset=utf-8")
```

Comment:
`application/json` defaults to `utf-8` so specifying the charset isn't really necessary. Additionally, can you add a `User-Agent` header here so we know where the request has come from?